autogen/oai/together.py

```diff
@@ -0,0 +1,351 @@
+"""Create a OpenAI-compatible client using Together. AI's API.
```

Comment:
```suggestion
"""Create an OpenAI-compatible client using Together. AI's API.
```

Reply:
Thanks, I've made an additional tweak to remove the space in "Together. AI". I'll commit it separately.